### ISSUE TITLE
resolution() blocks calc() for no benefit, but is a security boundary

### ISSUE THREAD

**Issue by alistair3149** [octfx/mediawiki-extensions-TemplateStylesExtender#62]
`MatcherFactoryExtender::resolution()` replaces upstream's implementation with a bare `TokenMatcher`, using a unit list (`dpi|dpcm|dppx|x`) identical to upstream's. So it adds nothing, and removes the `mathFunction()` wrapper upstream applies — meaning `image-set( url(…) calc(1x * 2) )` is refused while every other dimension slot in this extension accepts `calc()`.

Deleting the override would be a small, safe widening and would remove one more method shadowing upstream.

### Why it is not a simple deletion

The same override is what stops a URL-allowlist bypass. Upstream builds resolution as `mathFunction( rawResolution() )`, and `mathFunction()` is late-bound to this extension's `var()`-aware override, so inheriting it admits `var()` into the density slot of `image-set()`:

```css
--r: 2x, "https://evil.example/tracker.png" 1x;
background-image: image-set( "<allowed>" var(--r) )
```

which the browser resolves to an image-set whose second entry is fetched on a 1x display. `doSanitize()` does not catch it: the payload is a bare string, which is neither a `url()` token nor a named external-resource function — and a bare string is exactly what `image-set()`'s first argument accepts as a URL.

`UrlPolicyConfigTest::testResolutionSlotRejectsSubstitution` pins this, and fails if the override is removed.

### What a fix needs to do

Allow math functions in the density slot **without** allowing `var()` there — for instance by keeping an override that wraps `rawResolution()` in something math-aware but var-free, rather than inheriting `mathFunction()` wholesale.

Note that relaxing this at `mathFunction()` instead does not work: upstream's own `calcSum()` admits `var()`, on the reasoning that `calc()` forces values to be numeric. That stops holding once the result lands in a slot where substitution is textual, as it does here.
